iwyu_test_util.py

```diff
@@ -92,6 +137,32 @@ def HasTarget(target):
         supported.append(m.group(1))
     return target in supported
 
+  @staticmethod
+  def UsingCXXstdlib(lib):
+    id_macro = {
+      'libcxx' : '_LIBCPP_VERSION',
+      'libstdcxx' : '__GLIBCXX__',
+      'msvc' : '_MSVC_STL_UPDATE', # https://github.com/microsoft/STL/wiki/Macro-_MSVC_STL_UPDATE
```

Comment:
I think it would look nicer with this comment on a separate line above `'msvc': ...`